.github/workflows/check_black.yml

```diff
@@ -0,0 +1,13 @@
+name: Check Black formatting
+
+on: [push, pull_request]
+
+jobs:
+  black:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Black Code Formatter
+        uses: lgeiger/black-action@master
+        with:
+          args: ". -l 79 --check"
```

Comment:
FWIW I think I'm going to stick with black's default of 88 on other projects to avoid dealing with this. @hdoupe convinced me longer lines are cool.

Reply:
@MaxGhenis I started with default, but didn't love it.  I think @rickecon also has opinions.